src/Table2Beta/Table.tsx

```diff
@@ -53,6 +54,9 @@ export interface Props {
   rowClassNameFn?: Function;
   noDataContent?: React.ReactNode;
   selectable?: boolean;
+  selectedRowsHeaderContentType?: { singular: string; plural?: string };
+  // should this be optional?
+  selectedRowsHeaderActions: Array<ActionInput>;
```

Comment:
I'd argue this should indeed be optional, since `selectable` is optional.

Reply:
True!